### ISSUE TITLE
[Feature] GZDOOM VR 4.10.0

### ISSUE THREAD

**Issue by ReverquestVR** [hh79/gzdoomvr#88]
### GZDoom version

GZDOOM VR 4.9.0

### Which game are you running with GZDoom?

Other

### What Operating System are you using?

Windows 11

### Is your feature request related to a problem? Please describe.

No. We need versión 4.10.0 gzdoom in VR. Thanks

### Describe the solution you'd like

Create and public Gzdoom VR 4.10.0

**Comment by ni1chigo2115** [comment]
VKDoom is now available, still in the testing phase, but if you are going to use VR forks, this looks better from a PERFORMANCE standpoint.

**Comment by ReverVR** [comment]
> VKDoom is now available, still in the testing phase, but if you are going to use VR forks, this looks better from a PERFORMANCE standpoint.

VKDOOM dont support VR. Interested in VR support

**Comment by guitardz** [comment]
hh seems to have disappeared, Eviternity II has released and 4.9.0 can't load it because of it's advanced MBF21 dehackery, hh, please come back to us and update to the latest GZDoom version.

**Comment by knowsshit** [comment]
I hope someone knows how to backport GZDoom 4.11.1 or newer to GZDoom... If anyone has any clues on what needs to be done, maybe we can find a way.

**Comment by hh79** [comment]
Done, Eviternity II works fine.  https://github.com/hh79/gzdoomvr/releases/tag/gvr4.11.3

Now I realized I didn't release 4.10 version. It was merged and tested but I  somehow forget to make a release. The thing is, in my free time I prefer to play Doom than to hack it, so the glacial pace of progress.  However I intend to keep updating GZDoomVR till it would be technically possible. The OpenGL support is going to be dropped soon from GZDoom and I definitely have no skills to port VR code to Vulcan backend.
